cmake/CMakeLists.txt

```diff
@@ -902,6 +902,8 @@ if (CPUINFO_SUPPORTED)
   add_subdirectory(external/pytorch_cpuinfo EXCLUDE_FROM_ALL)
   if (MSVC)
     target_compile_options(cpuinfo PRIVATE "-D_CRT_SECURE_NO_WARNINGS")
+  else()
+    target_compile_options(cpuinfo PRIVATE "-Wno-stringop-overread")
```

Comment:
if you want to disable it, probably should check that it is available to disable. e.g.:
https://github.com/microsoft/onnxruntime/blob/3c967601921093ebffcdbed0a1baf1dc9f8b8f3d/cmake/CMakeLists.txt#L1002
and perhaps update this file so that info is available in code:
https://github.com/microsoft/onnxruntime/blob/3c967601921093ebffcdbed0a1baf1dc9f8b8f3d/cmake/onnxruntime_config.h.in